### PR TITLE
Refactor play loop

### PR DESCRIPTION
**Pull request by declanslevin**

### CODE REVIEW

File: lib/store.test.js [hunk 1]
```diff
@@ -51,8 +50,8 @@ describe("Store methods can set values", () => {
   });
 
   it("setCpuGridChoice stores grid choice of 5", () => {
-    const store = new Store();
-    store.cpuTeam = "X";
+    const store = new Store((cpuTeam = "X"));
```
Comment (alsuren) [inline lib/store.test.js]: I'm not familiar with this syntax in js. Do you mean
```suggestion
    const store = new Store({cpuTeam: "X"});
```
?

**Review by alsuren — APPROVED** [review]
This feels like a nicer shape. I would say it's ready for a websockets mode 😁

File: lib/store.test.js (hunk 1 above)
Comment (declanslevin) [reply lib/store.test.js]: I thought I needed to do it that way too, but my IDE keeps changing it automatically... it seems to work with () and = so I left it that way

**Comment by declanslevin** [comment]
I'm going to fix up the tests for the changes I made and then I'll get on to it

File: lib/store.test.js (hunk 1 above)
Comment (declanslevin) [reply lib/store.test.js]: Turns out this syntax might not work at all 🤔

**Comment by alsuren** [comment]
> I'm going to fix up the tests for the changes I made and then I'll get on to it

(probably something for a separate PR): It turns out that setting up github actions is for nodejs to run your tests is pretty quick

I went to https://github.com/alsuren/o-and-xs/actions/new (for you, it would be https://github.com/declanslevin/o-and-xs/actions/new) and in the `Workflows made for your JavaScript repository (Suggested)` section, it had a Node.js starter workflow that runs `npm test` (for every push to master and every pull request).
Screenshot 2021-01-13 at 10 20 25

It looks like it wouldn't be too difficult to make it run your project's tests (it also runs everything on a bunch of different node versions, but you can collapse that down to just one if you want to speed things up). I can also point you to a vscode plugin from red-hat that makes yaml editing less painful.

**Pull request merged**